src/main/java/pixelperfect/Game.java

```diff
@@ -43,6 +43,16 @@ public void simpleInitApp() {
 
     // Set up scenegraph.
   }
+  
+  /**
+   * Stops the game if the ship is dead
+   * 
+   */
+  public void deadShip() {
```

Comment:
This method should be called in the simpleUpdate method, otherwise the game is never stopped.